test/Interop/Cxx/value-witness-table/witness-layout-opts-irgen.swift

```diff
@@ -0,0 +1,24 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+
+// This test checks that the compiler uses well-known witness tables for types
+// that have the appropriate size/shape (for example, { i8 x 4 } -> i32).
+
+// These witness tables look very differnt on Windows so it doesn't make sense
+// to test them in the same file.
```

Comment:
As the comment says, the witness tables appear fairly different on Windows, so I don't think it's feasible to test both platforms in the same file. If anyone disagrees, let me know and I'll investigate further / update the test.